packages/hyperion-autologging/src/ALSurface.ts

```diff
@@ -282,13 +283,23 @@ export function init(options: InitOptions): ALSurfaceHOC {
       }, [domAttributeName, domAttributeValue]);
     }
 
+    ReactModule.useLayoutEffect(() => {
```

Comment:
see the code at line 273. We should be able to merge these code for calculating the value of `element`. 
If `ref` is there, element should b read from the `ref` and it is null, skip the rest of the code. 
Otherwise, the above code should work.